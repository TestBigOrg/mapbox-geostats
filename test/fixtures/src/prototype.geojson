{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "constructor": 2,
        "prototype": "foo"
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          84.72656249999999,
          55.7765730186677
        ]
      }
    },
    {
      "type": "Feature",
      "properties": {
        "constructor": 1
      },
      "geometry": {
        "type": "LineString",
        "coordinates": [
          [
            84.462890625,
            55.32914440840507
          ],
          [
            86.0009765625,
            55.751849391735284
          ],
          [
            86.044921875,
            56.04749958329888
          ]
        ]
      }
    }
  ]
}
