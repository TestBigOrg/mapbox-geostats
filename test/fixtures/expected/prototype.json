{
  "attributeCount": 1,
  "attributes": [
    {
      "attribute": "prototype",
      "count": 2,
      "max": 2,
      "min": 1,
      "type": "Number",
      "values": [
        1,
        2
      ]
    }
  ],
  "count": 2,
  "geometry": "Point",
  "layer": "prototype"
}
